lib/3scale/client.rb

```diff
@@ -132,8 +135,9 @@ def authrep(options)
     #                    from the UTC. For example, "US Pacific Time" has offset -0800, "Tokyo"
     #                    has offset +0900. This parameter is optional, and if not provided, equals
     #                    to the current time.
-    #   service_id::   ID of the service. It is optional. When not specified, the transactions
-    #                  are reported to the default service.
+    #   service_id::     ID of the service. It is optional. When not specified, the transactions
```

Comment:
Nitpick: shouldn't `service_id` and `service_token` be aligned with `transactions`? :)